CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 0.7.8
 -----
 
+* Backported PR#2019 to fix issue #2018
```

Comment:
This should actually describe the fix so people who install the update know what's changed.